D1673/P1673.md

```diff
@@ -3957,7 +3959,47 @@ on constraints wording.
 * D. Vandevoorde and N. A. Josuttis,
     "C++ Templates: The Complete Guide," Addison-Wesley Professional, 2003.
 
-# Wording
+# Dummy Heading To Align Wording Numbering
+
+# Dummy Heading To Align Wording Numbering
+
+# Dummy Heading To Align Wording Numbering
+
+# Dummy Heading To Align Wording Numbering
+
+# Dummy Heading To Align Wording Numbering
+
+# Dummy Heading To Align Wording Numbering
+
+# Dummy Heading To Align Wording Numbering
+
+# Dummy Heading To Align Wording Numbering
+
+# Dummy Heading To Align Wording Numbering
+
+# Dummy Heading To Align Wording Numbering
+
+# Dummy Heading To Align Wording Numbering
+
+# Dummy Heading To Align Wording Numbering
+
+## Dummy Heading To Align Wording Numbering
+
+## Dummy Heading To Align Wording Numbering
+
+## Dummy Heading To Align Wording Numbering
+
+## Dummy Heading To Align Wording Numbering
+
+## Dummy Heading To Align Wording Numbering
+
+## Dummy Heading To Align Wording Numbering
+
+## Dummy Heading To Align Wording Numbering
+
+The preceding headings just push the automatic numbering of the document generator so that "Wording is in the place of 28.8 Numbers
```

Comment:
It looks like there's a missing double quote after "Wording."

```suggestion
The preceding headings just push the automatic numbering of the document generator so that "Wording" is in the place of 28.8 Numbers
```